### DISCUSSION TITLE
Setting the Wii Remote & Nunchuks

### DISCUSSION THREAD

**Discussion by Hellghast** [Medard22/Dolphin-MMJR2-VBI#321] [Bug Report]
I reviewed all the settings and did not find the opportunity to configure Hotkeys. This is about the Sideways Toggle and Upright Toggle. Without them, it is impossible to play with the gamepad in the games Wii. What do you say about this, there are suggestions?

**Comment by Medard22** [comment]
MMJR2 uses an older input system. You can find the Sideways and Upright toggle in the Quick Menu-> Choose Controller. There is an option to use Horizontal Wii Remote, Wii Remote or Wii Remote + Nunchuk. 
Dolphin developers completely rewritten the input handling in Dolphin Official. However it makes massive changes to JNI and Android code that I wasn´t able to resolve. Unless I get help from another developer that is familiar with Android, there is not much I can do.

**Reply by ariel-tota5-madril** [reply]
im trying to get the tilting motion to work for games like need for speed pro street using the sticks to simulate it, but i cant seem to get it to work properly. I tried setting up the tilt forward/backward, left/right, etc, in both tilt, shake and any other option that reseambled motion simulation. How do i go about this?

**Reply by Medard22** [reply]
Try to map the motion controls to the analog stick and then in Overlay controls Disable motion input. Dolphin may try to use your device gyro instead of the mapped input.
